tests/Behat/Context/FeatureContext.php

```diff
@@ -16,39 +16,77 @@ class FeatureContext extends SilverStripeContext
      */
     public function iShouldSeeAListOfVersions()
     {
-        $page = $this->getSession()->getPage();
-        $versionsTable = $page->find('css', '.history-viewer .table');
-        assertNotNull($versionsTable, 'I should see a list of versions');
+        $this->getVersions();
     }
 
     /**
      * @Then I should see a list of versions in descending order
      */
     public function iShouldSeeAListOfVersionsInDescendingOrder()
     {
-        $page = $this->getSession()->getPage();
-        $versions = $page->findAll('css', '.history-viewer .table tbody tr td:nth-of-type(1)');
+        $versions = $this->getVersions();
+        $this->clickVersion(current($versions));
+    }
+
+    /**
+     * @When I click on the first version
+     */
+    public function iClickOnTheFirstVersion()
+    {
+        assertNotNull($this->getLatestVersion(), 'I should see a list of versions');
+        $this->getLatestVersion()->click();
+        // Wait for the form builder to load
+        $this->getSession()->wait(3000);
```

Comment:
This can be done more expressively in gerkin: `I wait for 3 seconds`

Reply:
True, I thought we'd actually replaced this with a javascript condition to match the form builder. I didn't want to pollute the user stories with statements like this, since the human doesn't really concern themselves with it - I guess though it is true, they will have to wait for the form builder to load...